tools/Makefile

```diff
@@ -154,22 +153,22 @@ endif
 # dependency for tools which have patches directory
 $(foreach tool, $(tools-y), $(if $(wildcard $(curdir)/$(tool)/patches),$(eval $(curdir)/$(tool)/compile += $(curdir)/patch/compile)))
 
-$(foreach tool, $(filter-out xz,$(tools-y)), $(eval $(curdir)/$(tool)/compile += $(curdir)/xz/compile))
+$(foreach tool, $(filter-out zstd,$(tools-y)), $(eval $(curdir)/$(tool)/compile += $(curdir)/zstd/compile))
 
 # make any tool depend on the following to ensure that archives can be unpacked and patched properly
 tools-core += libdeflate
 tools-core += patch
 tools-core += tar
-tools-core += xz
```

Comment:
We can't just drop xz, as many upstream tarballs are xz compressed, and we use xz to compress the initramfs if configured. So we still need it.

Reply:
Its not dropped, its just moved to be built later, in the same priority that ZSTD currently is, look for the line 72 diff

Reply:
Misread that part a bit, but AFAICT we still need it here else we become unable to extract .tar.xz tarballs until xz got built, also likewise the above where every tool depends on xz/compile.

Reply:
Well, it seems its still being built first like it used to.
I get your point though, its still required for all of the XZ compressed tarballs that are tool sources